packages/frontend-design-poc/src/utils/url.ts

```diff
@@ -0,0 +1,28 @@
+const LOGIN_REDIRECT_STORAGE_KEY = 'arbeidsflate::auth::login_Redirect';
```

Comment:
Can we call this file and folder something else than `utils/url.ts`? Maybe just call the folder for `auth`?